packages/ndla-ui/src/locale/messages-en.ts

```diff
@@ -338,13 +338,21 @@ const messages = {
       film: 'NDLA Film',
       about: {
         title: 'About NDLA',
-        whatIs: 'What is NDLA?',
-        organization: 'The organization',
-        numbers: 'Numbers and usage',
+        numbers: 'Numbers and reports',
         keyPersonnel: 'Key personnel',
         vacancies: 'Vacancies',
-        newsletter: 'Newsletter',
-        contact: 'Contact',
```

Comment:
Burde kanskje ikke slette disse på de ulike og heller ha de med videre for å ikke gjøre noe breaking?